package.json

```diff
@@ -1453,6 +1453,12 @@
                     "description": "Enables/disables A/B tests.",
                     "scope": "machine"
                 },
+                "jupyter.enableNativeInteractiveWindow": {
```

Comment:
How come we're not using the experiments framework for this? Is it because we don't need experiment data on this? It's really just a setting? 

Not sure if we still have those type of experiments. Experiments that are not backed by the experiment service.

Reply:
Yeah, this is just to target Insiders users for initial feedback, we're not going to rollout to Stable yet so experiment data isn't really necessary.